flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+	"only-arches": [
+		"x86_64"
```

Comment:
why? can't this be build from source?

Reply:
It can also be built for arm64, but I thought I would start with one arch and work my way up from there 🙂

Reply:
I'd start with building from source in the flatpak instead of unpacking a binary. This is the prefered way.

Reply:
I did start with that, it was unfortunately not attainable with my current level of skill.